lib/services/consumed/status-bar/status-bar.js

```diff
@@ -25,14 +25,16 @@ export function addStatusBar(
     priority: 100
   });
 
-  const onClick = showKernelCommands.bind(this, handleKernelCommand);
+  const onClick = _showKernelCommands.bind(this, handleKernelCommand);
```

Comment:
I worry about using `this`/`bind` outside of a class because I think it might be confusing to people who don't know the in's and outs of js `this`. Again, I wouldnt block on this but I think using either a class or: `let signalListView;` at the top of the file and referring to `signalListView` instead of `this.signalListView`

Reply:
Yeah I could probably class this too